### PR TITLE
fix(certs): add year to certs introduced in 2020

### PR DESCRIPTION
**Pull request by raisedadead**
Implements parts of #38609

### CODE REVIEW

File: client/src/components/settings/Certification.js [hunk 1]
```diff
@@ -482,8 +482,8 @@ export class CertificationSettings extends Component {
       is2018DataVisCert &&
       isApisMicroservicesCert &&
       isFrontEndLibsCert &&
-      isQaCert &&
-      isInfosecCert &&
+      is2020QaCert &&
+      is2020InfosecCert &&
       isJsAlgoDataStructCert &&
       isRespWebDesignCert;
```
Comment (raisedadead) [inline client/src/components/settings/Certification.js:488]: This block does not have the new python certs

**Review by raisedadead — COMMENTED** [review]
Needs investigation

**Comment by raisedadead** [comment]
Yikes - forgot to open this as draft. Apologies for the noise.

File: client/src/components/settings/Certification.js (hunk 1 above)
Comment (ojeytonwilliams) [reply client/src/components/settings/Certification.js:488]: Is the plan to make Full Stack include the three Python certs or are they going to be separate?

Comment (raisedadead) [reply client/src/components/settings/Certification.js:488]: IDK yet. I left an open issue to discuss with the team. 

Other than that this is a straight up /s/isCert/is2020Cert/g

Comment (raisedadead) [reply client/src/components/settings/Certification.js:488]: So, the current full stack will move into a legacy section.

**Review by ojeytonwilliams — APPROVED** [review]
If I try to claim the renamed certificates without seeding the database everything appears to work, even though, say, `is2020QaCert` is not present on the user record until that point.  I don't *think* this is a problem, but I thought I'd mention it.

**Comment by raisedadead** [comment]
> If I try to claim the renamed certificates without seeding the database everything appears to work, even though, say, `is2020QaCert` is not present on the user record until that point. I don't _think_ this is a problem, but I thought I'd mention it.

Yes, I believe this will be a expected issue in the local dev. I might purge the dev db before this gets deployed to the beta site. I will seed in some users from the production just so that we have a clean setup to find out any issues.

Let's hold on to this PR (for a quick bit) before I confirm all this.

**Pull request merged**